packages/functional-tests/tests/syncV3/signinCached.spec.ts

```diff
@@ -2,33 +2,41 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { expect, test, PASSWORD } from '../../lib/fixtures/standard';
+import {
+  expect,
+  test,
+  PASSWORD,
+  SIGNIN_EMAIL_PREFIX,
+  SYNC_EMAIL_PREFIX,
+} from '../../lib/fixtures/standard';
 
 test.describe('severity-2 #smoke', () => {
   test.describe('sync signin cached', () => {
     test.use({
-      emailOptions: [{ PASSWORD }, { prefix: 'sync{id}', PASSWORD }],
+      emailOptions: [
+        { prefix: SIGNIN_EMAIL_PREFIX, password: PASSWORD },
+        { prefix: SYNC_EMAIL_PREFIX, password: PASSWORD },
+      ],
     });
-    test.beforeEach(async ({ emails, target }) => {
+    test.beforeEach(async () => {
       test.slow(); //This test has steps for email rendering that runs slow on stage
-      const [email, syncEmail] = emails;
-      await target.auth.signUp(syncEmail, PASSWORD, {
-        lang: 'en',
-        preVerified: 'true',
-      });
-      await target.auth.signUp(email, PASSWORD, {
-        lang: 'en',
-        preVerified: 'true',
-      });
     });
 
     test('sign in on desktop then specify a different email on query parameter continues to cache desktop signin', async ({
       emails,
       target,
       syncBrowserPages: { page, login, connectAnotherDevice },
     }) => {
-      const [email, syncEmail] = emails;
       test.fixme(true, 'test to be fixed, see FXA-9194');
+      await Promise.all(
+        emails.map(async (email) => {
+          target.auth.signUp(email, PASSWORD, {
```

Comment:
Same Q here and on L83-84 as way above re: non-await `Promise.all()` w/ async `.map()`.

Reply:
Yep! will fix!